jupyter_server/serverapp.py

```diff
@@ -501,36 +475,17 @@ def start(self):
                 url = serverinfo['url']
                 if serverinfo.get('token'):
                     url = url + '?token=%s' % serverinfo['token']
-                print(url, "::", serverinfo['notebook_dir'])
+                print(url, "::", serverinfo['root_dir'])
 
 #-----------------------------------------------------------------------------
 # Aliases and Flags
 #-----------------------------------------------------------------------------
 
 flags = dict(base_flags)
-flags['no-browser']=(
-    {'NotebookApp' : {'open_browser' : False}},
-    _("Don't open the notebook in a browser after startup.")
-)
-flags['pylab']=(
-    {'NotebookApp' : {'pylab' : 'warn'}},
-    _("DISABLED: use %pylab or %matplotlib in the notebook to enable matplotlib.")
-)
-flags['no-mathjax']=(
-    {'NotebookApp' : {'enable_mathjax' : False}},
-    """Disable MathJax
-    
-    MathJax is the javascript library Jupyter uses to render math/LaTeX. It is
-    very large, so you may want to disable it if you have a slow internet
-    connection, or for offline use of the notebook.
-    
-    When disabled, equations etc. will appear as their untransformed TeX source.
-    """
-)
 
```

Comment:
Not Supporting the `no-mathjax` and `no-browser` flags anymore at the moment.

These should probably be added in the inheriting classes.